pom.xml

```diff
@@ -0,0 +1,483 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+    http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.openmicroscopy</groupId>
+  <artifactId>ome-codecs</artifactId>
+  <version>6.0.0-SNAPSHOT</version>
```

Comment:
Two thoughts here. For our immediate integration target, my preference is to release a 0.1.0 version which we can consume and wrap in Bio-Formats. That way if anything needs to be discussed, modified and/or adjusted, we can iterate quickly over `0.x.0` releases.

In the mid-run, once the integration is complete, we should definitely look into our roadmap towards a stable `1.0.0` for each of these decoupled components. I would drive this decision via our weekly meetings and make sure we collectively agree on whether we are happy with the state of the API as it stands or there is any breaking change we will have to consider in the next year or so.